src/ngx_http_lua_socket_tcp.c

```diff
@@ -316,6 +322,19 @@ ngx_http_lua_inject_socket_tcp_api(ngx_log_t *log, lua_State *L)
 
     lua_pushvalue(L, -1);
     lua_setfield(L, -2, "__index");
+
+#if (NGX_HTTP_SSL)
+
+#ifndef NGX_LUA_NO_FFI_API
+
+    /* expose tcp object metatable to global for FFI */
+    lua_pushvalue(L, -1);
+    lua_setglobal(L, ngx_http_lua_ngx_socket_tcp_mt_key);
```

Comment:
maybe we'd better set it into REGISTRY table? And we can get it by `debug.getregistry`.
like: https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/ctx.lua#L12